Whiskey/en-US/about_Whiskey_Writing_Tasks.help.txt

```diff
@@ -208,6 +208,16 @@ LONG DESCRIPTION
 
         [Whiskey.Task('TASK_NAME',Aliases=('DeprecatedName'),WarnWhenUsingAlias=$true)]
 
+    # Deprecating Tasks
+
+    To deprecate a task, set the `Obsolete` property on its `Whiskey.Task` attribute:
+
+        [Whiskey.Task('TASK_NAME',Obsolete)]
+
+    When someone uses your task, Whiskey will write a warning that the task is obsolete and shouldn't be used. You can customize the message shown to users with the `ObsoleteMessage` property:

```

Comment:
Nit: Doesn't `obsolete` imply that something is old/out-of-date because something newer/better is available? This may not always be the case when deprecating a task, a task may simply be retired without any replacement. Would `deprecated` be a better property name?

Reply:
`Obsolete` is what .NET uses. There may *not* be a new/better way. Usually there is. But deprecated is too specific. Obsolete leaves it open as to why someone shouldn't use a task anymore.